### PR TITLE
Transfer learning script additions: confusion matrix, `--path_out`, `--path_annotations_train`

### PR DESCRIPTION
**Pull request by guillaumebrg**

### CODE REVIEW

File: realtimenet/finetuning.py [hunk 1]
```diff
@@ -1,7 +1,11 @@
 import os
 import glob
+import itertools
+import json
 
+import matplotlib.pyplot as plt
 import numpy as np
+from sklearn.metrics import confusion_matrix
```
Comment (guillaumebrg) [inline realtimenet/finetuning.py:8]: TODO: edit dependencies

File: realtimenet/finetuning.py [hunk 2]
```diff
@@ -196,4 +211,41 @@ def run_epoch(data_loader, net, criterion, optimizer=None, use_gpu=False):
...
+    thresh = cm.max() / 2.
+    for i, j in itertools.product(range(cm.shape[0]), range(cm.shape[1])):
+        plt.text(j, i, cm[i, j],
+                 horizontalalignment="center",
+                 color="white" if cm[i, j] > thresh else "black")
+
+    # plt.tight_layout()
+    plt.ylabel('True label')
+    plt.xlabel('Predicted label')
+    plt.savefig(os.path.join(path_out, 'confusion_matrix.png'), bbox_inches='tight')
```
Comment (antoinemrcr) [inline realtimenet/finetuning.py:248]: plt.savefig(os.path.join(path_out, 'confusion_matrix.png'), transparent=False, pad_inches=0.1, bbox_inches='tight', dpi = 300)

The dpi might improve the quality of the image

Comment (guillaumebrg) [reply realtimenet/finetuning.py]: Done. Worked well and fixed the misalignment issue

**Comment by antoinemrcr** [comment]
Peut etre doit on mettre a jour le readme avec les nouvelles options ?

**Comment by antoinemrcr** [comment]
Should we update the readme ?

**Comment by guillaumebrg** [comment]
I don't think it's necessary since this is only for most advanced users and the default behavior hasn't changed... I would be up for leaving it like that and document this better once we have a proper wiki. What do you think?

File: requirements.txt [hunk 3]
```diff
@@ -4,6 +4,8 @@ numpy                   ==1.16.2      # BSD
 opencv-python           ==3.4.3.18    # MIT
 queuelib                ==1.5.0       # BSD
 docopt                  ==0.6.2       # MIT
+matplotlib              ==3.3.2       # BSD
+sklearn                               # BSD
```
Comment (florianletsch) [inline requirements.txt:8]: `sklearn` version missing deliberately?

Comment (guillaumebrg) [reply requirements.txt]: Fixed it.

**Pull request merged**